src/styles/illustrations/light.ts

```diff
@@ -0,0 +1,103 @@
+import EmptyStateBackgroundImage from '@assets/images/empty-state_background-fade-light.png';
+import Abracadabra from '@assets/images/product-illustrations/abracadabra.svg';
+import BankArrowPink from '@assets/images/product-illustrations/bank-arrow--pink.svg';
+import BankMouseGreen from '@assets/images/product-illustrations/bank-mouse--green.svg';
+import BankUserGreen from '@assets/images/product-illustrations/bank-user--green.svg';
+import ConciergeBlue from '@assets/images/product-illustrations/concierge--blue.svg';
+import ConciergeExclamation from '@assets/images/product-illustrations/concierge--exclamation.svg';
```

Comment:
@chrispader rather than importing all these images, what if we only used the provider for the theme specific ones and left the rest using the old pattern?

Reply:
yea sure, wasn't sure if all illustrations are theme-dependent.

once we have a list of all theme-dependent illustrations, i can simply adapt the affected components

Reply:
i'd still move the files with the imports to `src/styles/illustrations`. wdyt?

Reply:
Right now it is looking like it's only the background image, though the list might grow when we do a final QA of the theme. Hb we assume it's only that one for now?

Reply:
I'll leave the hook and context as is then and just update the list of imports. I can keep the branch with all of these changes, in case there are more dynamic illustrations coming soon

Reply:
@grgia updated the PR accordingly :)